src/main/java/frc/robot/commands/balancing/TuningBalancerCommand.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands.balancing;
+
+import java.util.ArrayList;
+
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.subsystems.drivetrain.BaseDrivetrain;
+
+public class TuningBalancerCommand extends DefaultBalancerCommand {
+
+
+    ArrayList<Double> time = new ArrayList<Double>();
+    ArrayList<Double> pitch = new ArrayList<Double>();
+    ArrayList<Double> power = new ArrayList<Double>();
+
+    Timer timer;
+
+    public TuningBalancerCommand(BaseDrivetrain driveSubsystem) {
+        super(driveSubsystem);
+        timer.reset();
+        timer.start();
+    }
+    
+    @Override
+    public void execute(){
+        super.execute();
+        pitch.add(super.getPitch());
+        power.add(super.getPower());
+        time.add(timer.get());
+    }
+
+    @Override
+    public boolean isFinished(){
+        if(super.isFinished()){
```

Comment:
Seems like you don't need this method at all? These prints should really go in `end()` because you want them to run when the command ends, which means `isFinished()` should just return `super.isFinished()` and you can let inheritance handle it.